### ISSUE TITLE
Error for min_size 

### ISSUE THREAD

**Issue by shan-stat** [MouseLand/cellpose#280]
Hi, thanks for developing the excellent program.

I tried to use the min_size argument in models.Cellpose.eval() function, but I think it did not work.
Please show the following results. Even though a mask consists of 82 pixels and the min_size argument is set to 100, the mask is not removed. Could you check this issue? Thanks.

---------------------------------------------------------------------------------------------------------------

>>> masks, flows, styles, diams = model.eval(img, diameter=20, channels=channels, flow_threshold=0.7, cellprob_threshold=-2, resample=True, min_size=15)
processing 1 image(s)
time spent: running network 2.58s; flow+mask computation 2.25
estimated masks for 1 image(s) in 4.89 sec
>>>> TOTAL TIME 4.89 sec
>>> masks.max()
301
>>> len(np.where(masks==4)[0])
82
>>> len(np.where(masks==4)[1])
82
>>> 
>>> masks, flows, styles, diams = model.eval(img, diameter=20, channels=channels, flow_threshold=0.7, cellprob_threshold=-2, resample=True, min_size=100)
processing 1 image(s)
time spent: running network 2.58s; flow+mask computation 2.25
estimated masks for 1 image(s) in 4.88 sec
>>>> TOTAL TIME 4.88 sec
>>> masks.max()
301
>>> len(np.where(masks==4)[0])
82
>>> len(np.where(masks==4)[1])
82

---------------------------------------------------------------------------------------------------------------

**Comment by GFleishman** [comment]
Just curious why this issue was closed? I have some questions about min_size as well; though have not observed this specific problem.

**Comment by ashishUthama** [comment]
I notice the same issue - Explicitly set min_size is ignored for 2D (defaults to 15).

Traced this to the following call where min_size is not propagated to dynamics.compute_masks in the 2D case:
https://github.com/MouseLand/cellpose/blob/e559dae0f9bb055aa69b84310229b6f66ea2492f/cellpose/models.py#L652
